src/dmd/root/ctfloat.d

```diff
@@ -216,17 +216,21 @@ extern (C++) struct CTFloat
     }
 
     // Constant real values 0, 1, -1 and 0.5.
-    __gshared real_t zero;
-    __gshared real_t one;
-    __gshared real_t minusone;
-    __gshared real_t half;
-
-    @trusted
-    static void initialize()
-    {
-        zero = real_t(0);
-        one = real_t(1);
-        minusone = real_t(-1);
-        half = real_t(0.5);
+    static if (real.sizeof > 8)
+    {
+        __gshared const real_t zero = 0;
+        __gshared const real_t one = 1;
+        __gshared const real_t minusone = -1;
+        __gshared const real_t half = 0.5;
+    }
+
+    else
+    {
+        import dmd.root.longdouble : ld_zero, ld_one, ld_minusOne, ld_half;
+
+        __gshared const real_t zero = ld_zero;
```

Comment:
perhaps `alias zero = dmd.root.longdouble.ld_zero;` ?

Reply:
Obsolete. Restored the `initialize` function per kinke's request.